.github/workflows/build.yml

```diff
@@ -54,6 +54,10 @@ on:
           - WARNING
           - INFO
           - DEBUG
+      build_name:
+        description: "Build name (defaults to branch name)"
+        type: string
+        required: false
```

Comment:
Figured it's nice to have the option to specify a custom build name

Reply:
I like it. I think any DB schema that isn't the same as an existing repo branch will get deleted at the end of the week though, just noting

Reply:
Yeah it's a little bit of a downside. But still nice every once in a while to be able to run two builds from the same branch without much hassle